CMakeLists.txt

```diff
@@ -11,7 +11,13 @@ SET_PROPERTY(GLOBAL PROPERTY USE_FOLDERS ON)
 
 # Using C++ 11
 SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD_REQUIRED ON) # Required
+SET(CMAKE_CXX_EXTENSIONS ON) # with compiler extensions like gnu++11
 
+# Avoid conflicts with min and max functions
+IF(WIN32)
```

Comment:
does the conflict only occurs on Windows?

Reply:
I think so, and only when you include windows.h.

Reply:
ok, great